src/tests/ftest/pool/create_query.py

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/python
```

Comment:
FYI - we shouldn't add the shebang for files that aren't directly executable. But no harm done
```suggestion
```

Reply:
I was not aware of that.
Thanks for the info.

Reply:
It's a new change :) In short, if a file doesn't have `execute` permission, it probably shouldn't have a shebang